packages/bootstrap/lib/Wrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { ThemeManager } from "@kaizen/design-tokens"
+import { themeSwitcher, getCurrentTheme } from "@cultureamp/theme-switcher"
+
+// Polyfill for :focus-visible pseudo-selector
+// See: https://github.com/WICG/focus-visible
+import "focus-visible"
+
+// Standard base stylesheet used across Culture Amp products
+// See: https://github.com/necolas/normalize.css/
+import "normalize.css"
+
+// Global styles
+import "./globals.module.scss"
+
+// Supported / active face declarations
+import "./fonts.module.scss"
+
+// Initialise the ThemeManager, this is a singleton so this should not be redeclared
+export const themeManager = new ThemeManager(getCurrentTheme())
+themeSwitcher({ themeManager })
```

Comment:
I'd still prefer to not include this here. Nothing else in Kaizen expects to be able to call a Culture Amp Platform API, and I don't really want to introduce that.

It'll be weird in places like Storybook (or our Gatsby site for that matter)

Reply:
Time for `frontend-services` - let's pair on this? If I don't have a better place to put the theme management my preference is to leave it here to avoid double handling of work and comms when this component rolls out.

Reply:
I'm happy to pair on it asap, yeah.